sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob/BlobServiceClientJavaDocCodeSnippets.java

```diff
@@ -240,4 +246,48 @@ public void getAccountInfoWithResponse() {
         StorageAccountInfo accountInfo = client.getAccountInfoWithResponse(timeout, context).getValue();
         // END: com.azure.storage.blob.BlobServiceClient.getAccountInfoWithResponse#Duration-Context
     }
+
```

Comment:
It looks like all of our batching tests are batching just 2 operations.  Can we test with an N greater than 2 as well?